mapstruct-on-gradle/build.gradle

```diff
@@ -1,27 +1,23 @@
 plugins {
     id 'java'
-    id 'net.ltgt.apt' version '0.9'
+    id 'net.ltgt.apt' version '0.20'
 }
 
 repositories {
     mavenCentral()
     mavenLocal()
 }
 
-task wrapper(type: Wrapper) {
-    gradleVersion = '3.4'
-}
-
 ext {
     mapstructVersion = "1.3.0.Beta2"
 }
 
-sourceCompatibility = JavaVersion.VERSION_1_8
+sourceCompatibility = "11"
```

Comment:
Why should this be 11?

Reply:
Hi guys, thanks for replying, I will remove `.DS_Store` file. And about `sourceCompatibility = 11` it was as a way to have the example with the latest java. But if it was not good I can keep it as `sourceCompatibility = JavaVersion.VERSION_1_8`. So @filiphr  you please just let me know.

Reply:
Let's keep 1.8. We can add a comment that people can put 11 and it would work.

Reply:
sure I will

Reply:
+1 for these changes and getting this PR merged!